extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -252,7 +252,7 @@ public function execute() {
 			 * (domain), $this->mCityId is unknown (set to false in constructor)
 			 */
 			wfProfileIn( __METHOD__."-domaincache" );
-			$key = WikiFactory::getDomainKey( $this->mServerName . '/' . $this->langCode );
+			$key = WikiFactory::getDomainKey( rtrim( $this->mServerName . '/' . $this->langCode, '/' ) );
```

Comment:
why rtrim?

Reply:
WF domain entries dont have a trailing slash so adding one here breaks domain cache invalidation where no slash is used